packages/nuxt/src/app/composables/ssr.ts

```diff
@@ -14,7 +14,11 @@ export function useRequestHeaders (include?: any[]) {
   const event = useRequestEvent()
   const headers = event ? getRequestHeaders(event) : {}
   if (!include) { return headers }
-  return Object.fromEntries(include.map(key => key.toLowerCase()).filter(key => headers[key]).map(key => [key, headers[key]]))
+  return Object.fromEntries(include.map(key => {
+   // Testing codspeed 
+    const lowerKey = key.toLowerCase()
+    return [lowerKey, headers[lowerKey]]
+  }).filter(([_, value]) => value !== undefined))
```

Comment:
```suggestion
})
```
This would just return a list of all the headers where the empty headers would be undefined, right?